glib/sys/Cargo.toml

```diff
@@ -99,16 +99,6 @@ version = "2.80"
 [package.metadata.system-deps.glib_2_0.v2_82]
 version = "2.81"
 
-[package.metadata.system-deps.gobject_2_0]
-name = "gobject-2.0"
-version = "2.56"
-
-[package.metadata.system-deps.gobject_2_0.v2_58]
-version = "2.58"
-
-[package.metadata.system-deps.gobject_2_0.v2_62]
```

Comment:
Why are these removed?

Reply:
At some point I might have been making some experiments, but yes, this should stay, sorry about that.